backend/app/src/main/resources/messages.properties

```
@@ -1,2 +1,3 @@
 columns.displayPreference.not.found=Columns display preference not found
-columns.displayPreference.invalidViewName=Allowed values for view names are {0},{1},{2},{3}
\ No newline at end of file
+columns.displayPreference.invalidViewName=Allowed values for view names are {0},{1},{2},{3}
+processData.mandatoryColumnDisplayName=Error Code: PHLAT.100 - {0} is mandatory
```

```suggestion
processData.mandatoryColumnDisplayName=Error Code: {} - {0} is mandatory
```
Since we have taken this as constant in code, we should then avoid hard coding here and substitute from code